setup.cfg

```diff
@@ -29,9 +30,7 @@ install_requires =
     pyyaml
     requests
     importlib-resources>=1.3;python_version<"3.9"
-    pathlib2>=2.1.0;python_version<"3"
-    typing;python_version<"3.5"
-python_requires = >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*
+python_requires = >=3.6
```

Comment:
It's actually more important, since lockfile resolvers and even normal pip installs don't handle caps correctly - this field was not designed to have limits. Numba had to remove their limit, in fact, even though they never support the next Python.

Reply:
Yeah it is slow going but I am trying to make a PR to every library out there that I come across in my personal usage that has a cap on Python version. Henry's blog post is powerful ammunition here (thanks!) for showing why this is so important, but even then it takes some arguing with people from time to time. :/